sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/externalrules/CxxExternalRulesSensor.java

```diff
@@ -32,22 +38,29 @@
 import org.sonar.plugins.cxx.CxxLanguage;
 import org.sonar.plugins.cxx.utils.CxxMetrics;
 import org.sonar.plugins.cxx.utils.CxxReportSensor;
+import org.sonar.plugins.cxx.utils.CxxUtils;
 import org.sonar.plugins.cxx.utils.StaxParser;
 
 /**
  * Custom Rule Import, all static analysis are supported.
  *
- * @author jorge costa
+ * @author jorge costa, stefan weiser
  */
 public class CxxExternalRulesSensor extends CxxReportSensor {
   public static final Logger LOG = Loggers.get(CxxExternalRulesSensor.class);
   public static final String REPORT_PATH_KEY = "sonar.cxx.other.reportPath";
+  public static final String SONAR_CXX_XSLT_KEY = "sonar.cxx.xslt.";
```

Comment:
I do agree with Stefan, this kind of transformation seems applicable only to external sensor since it's the only one that forces users to provide a specific format, @guwirth which other sensors do you think would benefit from this?